nodejs/nodejs-guestbook/kubernetes-manifests/guestbook-backend.deployment.yaml

```diff
@@ -36,3 +36,10 @@ spec:
               key: password
         - name: MONGO_HOST
           value: "mongo-service"
+        readinessProbe:
+          httpGet:
```

Comment:
probably the most important value you can customize is `periodSeconds`. it's 10s default. so if a check fails, it'll take 10s for another check to happen.

this is too slow for a keynote demo. make it like 3s. and make `timeoutSeconds: 1`. if the app can't connect to db within 1s, it's probably gonna magically succeed in the 2nd second.